src/tri.rs

```diff
@@ -30,38 +39,44 @@ where
     /// ```
     /// use ndarray::array;
     ///
-    /// let arr = array![[1, 2, 3], [4, 5, 6], [7, 8, 9]];
-    /// let res = arr.triu(0);
-    /// assert_eq!(res, array![[1, 2, 3], [0, 5, 6], [0, 0, 9]]);
+    /// let arr = array![
+    ///     [1, 2, 3],
+    ///     [4, 5, 6],
+    ///     [7, 8, 9]
+    /// ];
+    /// assert_eq!(
+    ///     arr.triu(0),
+    ///     array![
+    ///         [1, 2, 3],
+    ///         [0, 5, 6],
+    ///         [0, 0, 9]
+    ///     ]
+    /// );
```

Comment:
Ndarray functions try to document if they panic.

Here we need something like `**Panics** if k > something` (See other comments why, See other functions for the common format.)

Reply:
No longer panicking for any values of `k`, so I think we should be ok without a comment.